mscr-ui/src/modules/crosswalk-editor/tabs/metadata-and-files/index.tsx

```diff
@@ -1,173 +1,177 @@
+                        <div className='my-3'>PID: <span>{inputValue.pid}</span></div>
+                        <div className='my-3'>Version: <span>{inputValue.version}</span></div>
+                        <div className='my-3'>Created: <span>{inputValue.created}</span></div>
+                        <div className='my-3'>Modified: <span>{inputValue.modified}</span></div>
+                        <div className='my-3'>Publisher: <span>{inputValue.publisher}</span></div>
+                        <div className='my-3'>Creator: <span>{inputValue.creator}</span></div>
+                        <div className='my-3'>Contributors: <span>{inputValue.contributors}</span></div>
+                    </div>
+
+                    <div className='col-6'>
+                        <div className='mt-3'>
+                            <TextInput
+                                onChange={(value) => updateValue('notes', value)}
+                                labelText="Notes"
+                                value={inputValue.notes.toString()}
+                            />
+                        </div>
+                        <div className='mt-2'>
+                            <TextInput
+                                onChange={(value) => updateValue('description', value)}
+                                labelText="Description"
+                                value={inputValue.description.toString()}
+                            />
+                        </div>
+                        <div className='mt-2'>
+                            <TextInput
+                                onChange={(value) => updateValue('keywords', value)}
+                                labelText="Keywords"
+                                value={inputValue.keywords.toString()}
+                            />
+                        </div>
+                    </div>
+                </div>
             </div>
-          </div>
-        </div>
-        <h2 className="mt-4 mb-3">Files</h2>
-        <div className="row d-flex justify-content-between metadata-and-files-wrap mx-2">
-          <br />
+            <h2 className='mt-4 mb-3'>Files</h2>
+            <div className='row d-flex justify-content-between metadata-and-files-wrap mx-2'>
+                <br/>
 
-          <div>TABLE HERE</div>
-        </div>
-        <h2 className="mt-4 mb-3">Crosswalk summary</h2>
-        <div className="row d-flex justify-content-between metadata-and-files-wrap mx-2">
-          <JointListingAccordion
-            crosswalkJoints={props.crosswalks}
-            viewOnlyMode={true}
-            performAccordionAction={null}
-          ></JointListingAccordion>
+                <div>TABLE HERE</div>
+            </div>
+            <h2 className='mt-4 mb-3'>Crosswalk summary</h2>
+            <div className='row d-flex justify-content-between metadata-and-files-wrap mx-2'>
+                <JointListingAccordion crosswalkJoints={props.crosswalks} viewOnlyMode={true}
```

Comment:
This instance of the accordion should not have clickable texts, since the action it does is not applicable on this page